Tasks/AzureResourceGroupDeployment/package.json

```diff
@@ -1,9 +1,10 @@
 
 {
-  "name": "DeployAzureResourceGroup",
+  "name": "AzureResourceGroupDeployment",
   "main":"main.js",
   "dependencies": {
     "vsts-task-lib" : "^0.9.20",
-    "vso-node-api": "6.0.1-preview"
+    "vso-node-api": "6.0.1-preview",
+    "strip-json-comments": "1.0.4"
```

Comment:
This doesn't bring any dependencies. And yes, it's HIBS scanned. I have specifically used the one which was scanned